recipes/opentdf-client/all/conanfile.py

```diff
@@ -12,13 +18,10 @@ class OpenTDFConan(ConanFile):
     topics = ("opentdf", "opentdf-client", "tdf", "virtru")
     description = "openTDF core c++ client library for creating and accessing TDF protected data"
     license = "BSD-3-Clause-Clear"
-    generators = "cmake"
+    generators = "cmake", "cmake_find_package"
     settings = "os", "arch", "compiler", "build_type"
-    options = {"build_python": [True, False], "fPIC": [True, False]}
-    default_options = {"build_python": False, "fPIC": True}
-    exports_sources = ["CMakeLists.txt"]
-
-    _cmake = None
+    options = {"with_libiconv": [True, False], "with_zlib": [True, False], "with_libarchive": [True, False], "fPIC": [True, False], "branch_version": [True, False]}
+    default_options = {"with_libiconv": True, "with_zlib": True, "with_libarchive": False, "fPIC": True, "branch_version": False}
```

Comment:
```suggestion
    options = {"with_libarchive": [True, False], "fPIC": [True, False]}
    default_options = {"with_libarchive": False, "fPIC": True}
```
You should not cheat transitive dependencies. Instead, your dependency need to be updated. Now, Boost/1.79 is using the latest package versions, so it's no longer a problem for this recipe.

Reply:
Updated